automation/terraform/modules/o1-testnet/testnet.tf

```diff
@@ -28,16 +28,16 @@ module "kubernetes_testnet" {
   k8s_context    = var.k8s_context
   testnet_name   = var.testnet_name
 
-  use_local_charts   = false
+  use_local_charts   = true
```

Comment:
Can you update all of the chart versions in this PR and change this back to false before merging?